README.md

```diff
@@ -1 +1,32 @@
-# hw01
\ No newline at end of file
+# hw01
+
+My name is Dani Valverde. I'm a second-year in the College majoring in Statistics or Sociology, not sure which one yet. When I'm not in class, I'm involved in business RSOs on campus and I have an off-campus job coaching youth wrestling. I've lived in four different states and am moving to a fifth soon: 
+
+* Minnesota
+* Wisconsin
+* New York
+* Massachusetts
+* Arizona (moving soon!)
+
+Currently, I'm spending quarantine in Minnesota with my mom and my brother. 
+
+In quarantine, my most exciting kitchen adventure has been finding the **perfect edible cookie dough recipe**. I know to some people it might sound less than appetizing, but I *love* edible cookie dough. Here's the link to it: [https://livforcake.com/edible-cookie-dough-recipe/]
+
+
+Here's the ingredient list written out: 
+
+# Edible Cookie Dough
+
+* Heat-treated flour
+* Softened butter
+* Light brown sugar
+* Granulated sugar
+* Sea salt
+* Room temperature milk
+* Vanilla
+* Mini chocolate chips
+
+
+I figured out how to incorporate most of the elements of this assignment through accidentally starting the homework for my Stat 224 class first. One of the recommended files for the class is an R Markdown homework template, and the file demonstrated how to do most of the elements I needed. I also used this tutorial for figuring out the headers [https://rmarkdown.rstudio.com/articles_intro.html]. In terms of the Github workflow, I struggled the most with cloning the file and making sure it was in the right place on my computer. Pushing and committing was pretty straightforward when I did it for the setup, so I'm hoping it'll be the same now. 
+
+I had a lot of really scary experiences with R Markdown during an intro stat class I took in the fall, but it wasn't that scary during this assignment, which was a really nice surprise and makes me excited for the rest of the class! 
```

Comment:
It's not too complicated once you have some practice at it (and it helps when the focus of the class is not stats, but programming. And RMarkdown definitely uses programming)